build_minecraft_bot.sh

```diff
@@ -20,12 +20,21 @@ if [ -d catkin_ws/src/minecraft_bot ] ; then
 else
     cd catkin_ws/src
     # Add the package
-    ln -s ../../minecraft_bot/ minecraft_bot
+    catkin_create_pkg minecraft_bot std_msgs rospy roscpp
+    cp -R ../../minecraft_bot .
+    # ln -s ../../minecraft_bot/ minecraft_bot    Symlinks do not work in virtualbox natively
```

Comment:
If the Vagrantfile is particular to this project would you pls include it in the pr?